docs/plans/claudeops/claude-sdk-patterns.docs.md

```
@@ -0,0 +1,1586 @@
+
+interface NotificationEvent {
+  timestamp: string;
+  title?: string;
+  message: string;
+  sessionId: string;
+}
+```
+
+## Best Practices Summary
+
+### 1. Configuration Management
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)